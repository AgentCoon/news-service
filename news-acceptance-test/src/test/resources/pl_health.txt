{
  "status": "ok",
  "totalResults": 20,
  "articles": [
    {
      "source": {
        "id": null,
        "name": "Wprost.pl"
      },
      "author": null,
      "title": "Lekarz po dwóch minutach odesłał go ze szpitala. Nazajutrz pacjent zmarł",
      "description": "Trafił do szpitala z silnym bólem w klatce piersiowej. Lekarz, po dwóch minutach, bez badań odesłał go do domu. Pan Bogusław zmarł następnego dnia w drodze do szpitala.",
      "url": "https://www.wprost.pl/kraj/10106677/lekarz-po-dwoch-minutach-odeslal-go-ze-szpitala-nazajutrz-pacjent-zmarl.html",
      "urlToImage": "https://www.wprost.pl/_thumb/68/04/66d9eb834a31d28a455240156044.jpeg",
      "publishedAt": "2018-02-24T19:53:00Z"
    },
    {
      "source": {
        "id": null,
        "name": "Naszemiasto.pl"
      },
      "author": "BOK",
      "title": "Poznańscy seniorzy będą czekać krócej na operacje zaćmy",
      "description": "&nbsp;",
      "url": "http://lubon.naszemiasto.pl/artykul/poznanscy-seniorzy-beda-czekac-krocej-na-operacje-zacmy,4529744,art,t,id,tm.html",
      "urlToImage": "https://d-nm.ppstatic.pl/kadr/k/r/1a/cf/5a91a39425271_o,size,610x0,q,70,h,244f1d.jpg",
      "publishedAt": "2018-02-24T18:28:40Z"
    }
  ]
}