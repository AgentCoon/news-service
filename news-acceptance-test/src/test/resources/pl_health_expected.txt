{"country":"pl","category":"health","articles":[{"author":null,"title":"Lekarz po dwóch minutach odesłał go ze szpitala. Nazajutrz pacjent zmarł","description":"Trafił do szpitala z silnym bólem w klatce piersiowej. Lekarz, po dwóch minutach, bez badań odesłał go do domu. Pan Bogusław zmarł następnego dnia w drodze do szpitala.","date":"2018-02-24","sourceName":"Wprost.pl","articleUrl":"https://www.wprost.pl/kraj/10106677/lekarz-po-dwoch-minutach-odeslal-go-ze-szpitala-nazajutrz-pacjent-zmarl.html","imageUrl":"https://www.wprost.pl/_thumb/68/04/66d9eb834a31d28a455240156044.jpeg"},{"author":"BOK","title":"Poznańscy seniorzy będą czekać krócej na operacje zaćmy","description":"&nbsp;","date":"2018-02-24","sourceName":"Naszemiasto.pl","articleUrl":"http://lubon.naszemiasto.pl/artykul/poznanscy-seniorzy-beda-czekac-krocej-na-operacje-zacmy,4529744,art,t,id,tm.html","imageUrl":"https://d-nm.ppstatic.pl/kadr/k/r/1a/cf/5a91a39425271_o,size,610x0,q,70,h,244f1d.jpg"}]}